### ISSUE TITLE
Latest update causes `error: failed to push some refs to git@gitlab....`

### ISSUE THREAD

**Issue by alexjking** [cmderdev/cmder#1124]
The latest update this morning causes the error `error: failed to push some refs to 'git@...'` when running `git push`.  The push is successful however this error displays despite the push being successful. ConEmu has been updated to build 161009. Could this be the issue here?

**Comment by geancanova** [comment]
it is not identifying pulls too

**Comment by doubleUTF** [comment]
I get a similiar error when I try `git push` as well. 
```
Username for 'https://github.com': doubleutf error: failed to execute prompt script (exit code 5) fatal: 
could not read Username for 'https://github.com': Invalid argument
```

**Comment by jrajav** [comment]
Piggybacking on this to say that the latest update also breaks my msys2 bash task `%ConEmuDrive%\msys64\usr\bin\bash.exe --login -i -new_console:C`. Bash starts up but acts oddly - the prompt is the default one, tab completion isn't enabled, other issues. It seems like its environment isn't being set up properly.

**Comment by au** [comment]
Since people's environments have been breaking since the new update, try a solution I found for #1123, which has had a similar problem with improper bash functionality.

Disable %ConEnuBaseDir% in your environment variables (Startup > Environment), and try again.

**Comment by Stanzilla** [comment]
Fixed in 161009a https://github.com/Maximus5/ConEmu/releases/tag/v16.10.09a
